From: Mike Kasick <mike@kasick.org>
Date: Tue, 08 Nov 2011 11:40:02 -0500

Change initramfs to be more like a userdebug ROM.

Set ro.debuggable=1 so "adb root" works, but doesn't default to root.
Start adbd service in recovery regardless of "USB debugging" setting.
Use testkey for update.zip signature verification.

diff --git a/initramfs/default.prop b/initramfs/default.prop
--- a/initramfs/default.prop
+++ b/initramfs/default.prop
@@ -3,5 +3,5 @@
 #
 ro.secure=1
 ro.allow.mock.location=0
-ro.debuggable=0
+ro.debuggable=1
 persist.service.adb.enable=0
diff --git a/initramfs/fota.rc b/initramfs/fota.rc
--- a/initramfs/fota.rc
+++ b/initramfs/fota.rc
@@ -94,10 +94,3 @@
 service recovery /system/bin/recovery
 
 service adbd /sbin/adbd recovery
-    disabled
-
-#on property:persist.service.adb.enable=1
-#    start adbd
-
-#on property:persist.service.adb.enable=0
-#    stop adbd
diff --git a/initramfs/recovery.rc b/initramfs/recovery.rc
--- a/initramfs/recovery.rc
+++ b/initramfs/recovery.rc
@@ -94,10 +94,3 @@
 service recovery /system/bin/recovery
 
 service adbd /sbin/adbd recovery
-    disabled
-
-#on property:persist.service.adb.enable=1
-#    start adbd
-
-#on property:persist.service.adb.enable=0
-#    stop adbd
diff --git a/initramfs/res/keys b/initramfs/res/keys
--- a/initramfs/res/keys
+++ b/initramfs/res/keys
@@ -1,1 +1,1 @@
-{64,0x591f5d5d,{993403147,2102747760,2116423980,1520664856,3855775470,1301258913,603968903,889265256,600622491,1751845310,840609692,1573324314,4204631367,508645623,3865742445,2230892497,4036346813,108484070,441050656,879206535,2973613679,2996749531,1332196159,3693459404,295103755,1694818649,953021024,1565906065,1653217009,567027052,2823740415,388172039,667899318,4127285460,845857575,183315048,1364588227,3071942183,74204099,2251049126,1053270747,2187000657,1875226878,3308670371,488264117,1268587879,343694789,210601327,2650063028,2260915049,840915831,2239333325,2583858656,4012727203,1395493106,63991089,2524499563,1212266286,1137920117,2808525882,559541646,3462588047,606281757,3924946356},{3945628694,1185315627,1980939842,2613340636,3519415799,4068247955,1355350363,3713288919,3214203000,1730252395,1554951019,40926085,2963003407,276771666,2583056483,2350770080,137674418,4076947933,3345314490,2065505354,3629565996,2984671262,1340195370,2950982580,2190923881,1979384954,4073342993,1687469836,1648156331,836160036,1126134217,1363119325,2459679313,3628038041,2265540653,3479455905,3142221008,96265552,3921482948,3602791822,2530916870,1938154501,1409067644,1668428227,807082600,3779961757,3000930953,2196777208,416758534,3267905347,1192483377,2110404486,703280182,2396083583,743146620,3786184685,3159760427,382976856,4174184062,3703895702,681704461,1342389241,3001032107,3118363299}}
+{64,0xc926ad21,{1795090719,2141396315,950055447,-1713398866,-26044131,1920809988,546586521,-795969498,1776797858,-554906482,1805317999,1429410244,129622599,1422441418,1783893377,1222374759,-1731647369,323993566,28517732,609753416,1826472888,215237850,-33324596,-245884705,-1066504894,774857746,154822455,-1797768399,-1536767878,-1275951968,-1500189652,87251430,-1760039318,120774784,571297800,-599067824,-1815042109,-483341846,-893134306,-1900097649,-1027721089,950095497,555058928,414729973,1136544882,-1250377212,465547824,-236820568,-1563171242,1689838846,-404210357,1048029507,895090649,247140249,178744550,-747082073,-1129788053,109881576,-350362881,1044303212,-522594267,-1309816990,-557446364,-695002876},{-857949815,-510492167,-1494742324,-1208744608,251333580,2131931323,512774938,325948880,-1637480859,2102694287,-474399070,792812816,1026422502,2053275343,-1494078096,-1181380486,165549746,-21447327,-229719404,1902789247,772932719,-353118870,-642223187,216871947,-1130566647,1942378755,-298201445,1055777370,964047799,629391717,-2062222979,-384408304,191868569,-1536083459,-612150544,-1297252564,-1592438046,-724266841,-518093464,-370899750,-739277751,-1536141862,1323144535,61311905,1997411085,376844204,213777604,-217643712,9135381,1625809335,-1490225159,-1342673351,1117190829,-57654514,1825108855,-1281819325,1111251351,-1726129724,1684324211,-1773988491,367251975,810756730,-1941182952,1175080310}}
